WTWSMS/localisation/WtWSMS_cultures.csv

```diff
@@ -818,7 +817,7 @@ damnonii;Damnonii;damnonii;;;Dumnonia;;;;;;;x
 venedi;Venedi;wende;;;V�neda;;;;;;;x
 cantabrian;Cantabrian;cantabre;;;C�ntabra;;;;;;;x
 khwarezmian;Khwarezmian;chorasmien;;;Khwarezmia;;;;;;;x
-latin;Latin;latin;;;Latina;;;;;;;x
+#latin;Latin;latin;;;Latina;;;;;;;x
```

Comment:
Why did you disable this?

Reply:
duplicate with vanilla

Reply:
The localisation isn't for the group though, it is for the culture.

Reply:
It is intended for the Latin culture added by the mod though, not the Latin culture group from vanilla.

Reply:
Latin being a culture group is not a problem here. The looked at the localisation in vanilla again and the mod one is better. I will enable it.

Reply:
done